### ISSUE TITLE
OSERROR :couldnt fund valid .pth checkpoint under 'models/TuodMango'

### ISSUE THREAD

**Issue by sylvester252** [dusty-nv/jetson-inference#1625]
Hello after training my custom  datasheet and exporting it to onnx it give me an error ssting OSERROR: couldnt fund valid .pth checkpoint under 'models/TuodMango'

**Comment by dusty-nv** [comment]
@sylvester252 can you confirm that you have any .pth files under that directory?

**Comment by sylvester252** [comment]
Yes I found the problem and its working now thabk you for responding I saw
your video it really helped us.we are curre tly making our final thesis
project for the last requirement. I thank you for replying master dusty

**Comment by zulkhaireey** [comment]
Hye, i encountered the same problem. May i know how you solve this? Thanks
